Samples/Islands/DrawingIsland/DrawingIslandComponents/DrawingIsland.h

```diff
@@ -194,6 +196,8 @@ namespace winrt::DrawingIslandComponents::implementation
         {
             winrt::Compositor Compositor{ nullptr };
 
+            std::shared_ptr<TextRenderer> TextRenderer;
```

Comment:
You're initializing text as part of UIA? What if UIA is never instantiated? This seems odd.

Reply:
I don't know about UIA. I put the TextRenderer alongside the Compositor because it depends on the compositor and is used to create visuals for text items. It seemed like a reasonable fit, but I don't really understand the purpose of grouping the m_output members into a struct, so I could easily be wrong about that. I'm happy to move it if you can suggest another location.

Reply:
Never-mind. Looking at the code in GitHub was confusing. I got it successfully to load in CodeFlow, which is much better.